### ISSUE TITLE
Add a device K51 Collective

### ISSUE THREAD

**Issue by jnr4817** [Rexeh/joystick-diagrams#67]
Would you consider adding the K51 collective to the templates and/or program.

https://forum.dcs.world/topic/222296-sollective-pitch-control-lever/

**Comment by Rexeh** [comment]
Hi,

Do you have someone who are able to help with the button outs? That's the hardest part, if you know someone with the device and can give the button_ids in relation to stick it can be done!

Cheers

Rob

**Comment by jnr4817** [comment]
I own the device and am happy to help. What do you need?


Sent from Yahoo Mail for iPhone


On Sunday, March 20, 2022, 10:48, Robert Cox ***@***.***> wrote:




Hi,

Do you have someone who are able to help with the button outs? That's the hardest part, if you know someone with the device and can give the button_ids in relation to stick it can be done!

Cheers

Rob

—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by Rexeh** [comment]
1. Some drawing/mock-up/image of the device
2. Rough idea of what each switch registers as I.e. BUTTON_1, BUTTON_2

Looking at the device, it's similar to a virpil stick, so ideally this would be for the base firmware, rather than user-specific to yourself. In Virpil products, someone can have BUTTON_1 and another with same stick and same button is BUTTON_26 - because of the customisation allowed in firmware.

Basically need to do what is in the video here - https://www.youtube.com/watch?v=me88WVqPaFU&t=16s - The resolution is too low for me to figure out what is what :)

**Comment by Rexeh** [comment]
Closing this ticket, no further information provided. Requests can be made via the discord for new templates
